src/main/java/Service/MailApi.java

```diff
@@ -0,0 +1,82 @@
+package Service;
+import java.lang.*;
+import java.util.Properties;
+
+import com.sun.xml.internal.messaging.saaj.packaging.mime.MessagingException;
+import com.sun.xml.internal.org.jvnet.mimepull.MIMEMessage;
+
+import javax.mail.*;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+
+public class MailApi {
+
+
+
+    public static void email(String email) throws MessagingException {
+
+        String message = "Welcome,Thank you for the registration. Please don't reply this message for security purpose..Khushh rhooooooo";
+        String subject = "For confirmation";
+        String to = email;
+        String from = "bhavna.gawhade@hotwaxsystems.com";
+
+        sendEmail(message, subject, to, from);
+
+
+
+    }
+    // this method is responsible for send message
+    private static void sendEmail(String message, String subject, String to, String from) {
+        // variable for gmail
+        String host ="smtp.gmail.com";
+
+        //get the system properties
+        Properties  properties =System.getProperties();
+        System.out.println("Properties"+ properties);
+
+        //setting important information
+        properties.put("mail.smtp.host",host);
+        properties.put("mail.smtp.port","465");
+        properties.put("mail.smtp.ssl.enable","true");
+        properties.put("mail.smtp.auth","true");
+
+        //to get the session object
+        Session session = Session.getInstance(properties, new Authenticator() {
+            protected PasswordAuthentication getPasswordAuthentication(){
+
+                return  new PasswordAuthentication("bhavna.gawhade@hotwaxsystems.com","Bhavi12@#");
```

Comment:
This is a bad practice. Please don't use your credentials in the Java files. A few days ago I shared an option to do so: 
https://www.baeldung.com/java-properties
https://www.geeksforgeeks.org/reading-and-writing-properties-file-in-java/